Jenkinsfile

```diff
@@ -29,25 +30,16 @@ import com.datastax.jenkins.drivers.python.Slack
 
 slack = new Slack()
 
-// Define our predefined matrices
-//
-// Smoke tests are CI-friendly test configuration.  Currently-supported Python version + modern C*/DSE instances.
-// We also avoid cython since it's tested as part of the nightlies.
 DEFAULT_CASSANDRA = ['2.1', '2.2', '3.0', '3.11', '4.0']
 DEFAULT_DSE = ['dse-5.0.15', 'dse-5.1.35', 'dse-6.0.18', 'dse-6.7.17', 'dse-6.8.30']
-DEFAULT_RUNTIME = ['2.7.18', '3.5.9', '3.6.10', '3.7.7', '3.8.3']
+DEFAULT_RUNTIME = ['3.7.7', '3.8.3']
```

Comment:
Didn't seem to make much sense to include Python versions we no longer explicitly support